src/share/poudriere/common.sh

```diff
@@ -2514,17 +2514,35 @@ jail_start() {
 	fi
 	injail id >/dev/null 2>&1 || \
 	    err 1 "Unable to execute id(1) in jail. Emulation or ABI wrong."
+	portbuild_gid=$(injail pw groupshow ${PORTBUILD_GROUP} 2>/dev/null | cut -d : -f3 || :)
+	if [ -z "${portbuild_gid}" ]; then
+		msg_n "Creating group ${PORTBUILD_GROUP}"
+		injail pw groupadd ${PORTBUILD_GROUP} -g ${PORTBUILD_GID} || \
+		err 1 "Unable to create group ${PORTBUILD_GROUP}"
+		echo " done"
+	else
+		PORTBUILD_GID=${portbuild_gid}
+	fi
 	portbuild_uid=$(injail id -u ${PORTBUILD_USER} 2>/dev/null || :)
 	if [ -z "${portbuild_uid}" ]; then
-		msg_n "Creating user/group ${PORTBUILD_USER}"
-		injail pw groupadd ${PORTBUILD_USER} -g ${PORTBUILD_UID} || \
-		err 1 "Unable to create group ${PORTBUILD_USER}"
-		injail pw useradd ${PORTBUILD_USER} -u ${PORTBUILD_UID} -d /nonexistent -c "Package builder" || \
+		msg_n "Creating user ${PORTBUILD_USER}"
+		injail pw useradd ${PORTBUILD_USER} -u ${PORTBUILD_UID} \
```

Comment:
Since you're changing all of the code here can you please quote the params to `pw` to avoid file substitution?